core/src/chain/pair.rs

```diff
@@ -115,7 +108,7 @@ pub mod tests {
 
     /// dummy pair
     pub fn test_pair() -> Pair {
-        Pair::new(&test_chain(), &test_entry())
+        test_chain().create_next_pair(&test_entry())
```

Comment:
Given that you are renaming most uses of "pair" to "entry" wouldn't it be good to similarly change the names in the tests?  Or is there a reason for keeping it pair?